source/github-helpers/index.ts

```diff
@@ -37,6 +37,34 @@ export const getCurrentCommittish = (pathname = location.pathname, title = docum
 	return unslashedCommittish;
 };
 
+export const getCurrentBranchFromFeedLink = (): string | undefined => {
```

Comment:
I'm imagining this to be part of `getCurrentCommittish` itself since they do the same thing and they're both dom-ready-safe